src/components/MyOrdersWidget/subcomponents/Order/Order.tsx

```diff
@@ -80,8 +81,12 @@ const Order: FC<PropsWithChildren<OrderProps>> = ({
         <StyledTokenLogo size="tiny" tokenInfo={signerTokenInfo} />
         <StyledTokenLogo size="tiny" tokenInfo={senderTokenInfo} />
       </TokenLogos>
-      <Text>{`${signerAmount} ${signerTokenInfo?.symbol || ""}`}</Text>
-      <Text>{`${senderAmount} ${senderTokenInfo?.symbol || ""}`}</Text>
+      <Text>{`${getAbbreviatedNumber(signerAmount.toString())} ${
+        signerTokenInfo?.symbol || ""
+      }`}</Text>
+      <Text>{`${getAbbreviatedNumber(senderAmount.toString())} ${
+        senderTokenInfo?.symbol || ""
+      }`}</Text>
```

Comment:
Move the `getAbbreviatedNumber(` in the `const senderAmount` and `const signerAmount` `useMemo`.